### PR TITLE
[ONNX] Expose onnx_import C++ interface in Python API.

### PR DESCRIPTION
**Pull request by arogowie-intel**

### CODE REVIEW

File: python/pyngraph/onnx_import.cpp [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+#include <vector>
+#include "ngraph/frontend/onnx_import/onnx.hpp"
+#include "ngraph/function.hpp"
+
+namespace py = pybind11;
+
+static std::vector<std::shared_ptr<ngraph::Function>>
+    load_onnx_model(const std::string& model_proto)
+{
+    std::istringstream iss(model_proto,
```
Comment (apwojcik) [inline python/pyngraph/onnx_import.cpp]: Why `ate` - seek to the end of the stream?

File: python/pyngraph/onnx_import.cpp [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                           std::ios_base::binary | std::ios_base::in | std::ios_base::ate);
+    return ngraph::onnx_import::load_onnx_model(iss);
```
Comment (apwojcik) [inline python/pyngraph/onnx_import.cpp]: Why `std::ios_base::ate`? This means seek to the end.

Comment (arogowie-intel) [reply python/pyngraph/onnx_import.cpp]: My mistake, actually it's not necessary.

File: python/test/ngraph/test_onnx_import.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import numpy as np
+
+from ngraph.impl.onnx_import import onnx_import
+from test.ngraph.util import get_runtime
+
+def test_import_onnx_function():
+    runtime = get_runtime()
+    dtype = np.float32
+    cur_dir = os.path.dirname(__file__)
+    model_path = os.path.join(cur_dir, '../../../test/models/onnx/add_abc.onnx')
```
[the remark this answers is not in the document]
Comment (arogowie-intel) [reply python/test/ngraph/test_onnx_import.py]: Copied model to `python/test/ngraph/models`

File: python/ngraph/runtime.py [hunk 4]
```diff
@@ -47,48 +47,43 @@ def __init__(self, backend_name):  # type: (str) -> None
     def __repr__(self):  # type: () -> str
         return '<Runtime: Backend=\'{}\'>'.format(self.backend_name)
 
-    def computation(self, node, *inputs):  # type: (Node, *Node) -> 'Computation'
+    def computation_node(self, node, *inputs):  # type: (Node, *Node) -> 'ComputationNode'
```
[the remark this answers is not in the document]
Comment (arogowie-intel) [reply python/ngraph/runtime.py]: Reverted back API changes.

**Comment by postrational** [comment]
TESTNOW

**Pull request merged**